apps/wallet/src/features/Widget/BuiltInNetwork/CollectiblesTab.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
 
 export const CollectiblesTab: FC<Props> = ({ collections = [] }) => {
 	const { onGridContainerLayout, width } = useLazyGridLayout({
-		referenceWidth: 156,
+		referenceWidth: 150,
```

Comment:
wonder why need this change?

Reply:
The `width` returned from `useLazyGridLayout` in extension layout is always `151px`. The calculated width need to be smaller than the reference to keep the layout in CollectiblesTab has 2 columns, therefore we just have to pick a reference value that smaller than calculated. The responsiveness in small view is the same.